Readme.md

```diff
@@ -21,3 +21,10 @@ A second difference is that common open source software and specification licens
 ## Who developed the Community Specification
 
 The Community Specification has been developed via the [Joint Development Foundation](http://www.jointdevelopment.org), with inspiration from the [Open Web Foundation agreements](http://openwebfoundation.org) and the [Alliance for Open Media Patent License 1.0](http://aomedia.org/license/patent-license/).
+
+The SPDX Identifier for the Community Specification V1.0 is, [`Community-Spec-1.0`](https://spdx.org/licenses/Community-Spec-1.0.html).

```

Comment:
I might just insert the word "License" after "SPDX" and also after "Specification" here, otherwise looks good. Also there might be a stray comma after "is".